src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContext.cs

```diff
@@ -3,14 +3,17 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Reflection;
 
 namespace Microsoft.Extensions.DependencyModel
 {
     public class DependencyContext
     {
-        private static readonly Lazy<DependencyContext> _defaultContext = new Lazy<DependencyContext>(LoadDefault);
```

Comment:
Can you undo this change? The proposed change doesn't have the same semantics as the existing code - you can get 2 DependencyContext objects created.

Reply:
Ahhh yeah thread safety, I will suppress the warning on the field and add RequiresAssemblyFiles on the Default function